packages/components/src/components/input/input.scss

```diff
@@ -115,8 +115,32 @@
         cursor: text;
         font-size: var(--text-size-xs);
 
-        + .dc-input__field {
-            padding-left: calc(1.6rem + 6px);
+        &.symbols {
+            top: 0.9rem;
+
+            + .dc-input__field {
+                // for the default value put the largest padding
+                // in case any new currencies get added
+                padding-left: calc(1.6rem + 15px);
+            }
+            &--usd + .dc-input__field {
```

Comment:
@negar-binary how do you add this symbol classes to input? is it coming from app?

Reply:
yes, I put it in this file because most people will be using leading_icon for putting currency symbols inside input fields as per the design.